src/settings.py

```diff
@@ -52,6 +52,16 @@
 PROVIDER_ROLE = 'seller'
 CUSTOMER_ROLE = 'customer'
 
+CURRENCY_CODES = [
+    ('EUR', 'Euro',),
+]
+
+CHARGE_PERIODS = (
+    'monthly',
```

Comment:
Probably these constants should include the number of days the constant is referring to